docs/aggregator.md

```diff
@@ -48,6 +51,14 @@ List of supported fiat currencies can be found in CurrencyId enum of [proto defi
    }));
    ```
    Please check **EstimateOffRampFeeResponse** of [proto definitions](../proto/ramp/v1/public.proto) for detailed documentation over response data
+6. Open a popup window passing parameters
+
+   Once user selected Harbour as ramp provider user should be directed to Harbour ramp web app at
+   `https://.../?origin={metamask}&op={buy|sell}&amount=100&asset={ASSET_ID_USDC|ASSET_ID_ETH|...}&protocol={ETHEREUM|POLYGON|...}&currency={EUR}`
+   Please note that
+   * amount should be in `fiat` for `buy` operation and in 'crypto' for `sell` operation 
+   * `asset`, `protocol` and `currency` values should be the same as string representation of enum in [proto definitions](../proto/ramp/v1/public.proto)   
```

Comment:
What exactly is the string representation? Cause if it's `CURRENCY_ID_EUR` then it's contradicting the redirect example on line 57 which states `&currency={EUR}`